shell/platform/common/cpp/geometry.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Please add a unit test for this file. I know there's not much to test in Rect, but we should validate all the basic logic (all the trivial accessors actually return the right component of the rect, width/height are always positive, etc.), and having the file in place reduces the chance that people will add more things without tests later.

Reply:
Done.